### PR TITLE
Avoid to copy-paste `_package` folder

### LINKED ISSUE #106: Issue with `_package` template
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

The `_package` folder needs to be copy pasting to run the command `pyconverter-xml2py package  -x XML_directory_path`. 
This behavior should be modified. 

- In the case there is a `_package folder` in the current working directory: use this template. 
- Otherwise, use the one within `pyconverter-xml2py`.

### 📝 Steps to reproduce

Go to a clean working directory and run: 
```
pip install pyconverter-xml2py
pyconverter-xml2py package  -x XML_directory_path
```

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

-

### 🐍 Which Python version are you using?

3.10

### 📦 Installed packages

```shell
-
```

### PR DESCRIPTION
**Pull request by clatapie**
Closes #106.

### CODE REVIEW

File: src/pyconverter/xml2py/writer.py [hunk 1]
```diff
@@ -239,14 +240,17 @@ def write_source(
...
+    target_path : str
+        Path where to generate the new package.
 
     path_custom_functions : str, optional
         Path containing the customized functions. The default is ``None``.
 
-    new_package_path : str, optional
-        Path to copy the ``_package`` directory to. The default is ``./package``.
+    new_package_name : str, optional
+        The default is ``package``.
```
Comment (PipKat) [inline src/pyconverter/xml2py/writer.py:250]:
```suggestion
        Name of the new package. The default is ``package``.
```

File: src/pyconverter/xml2py/writer.py [hunk 2]
```diff
@@ -239,14 +240,17 @@ def write_source(
...
+    
+    template_path : str, optional
+        If not provided, the default template will be the default one.
```
Comment (PipKat) [inline src/pyconverter/xml2py/writer.py:253]:
```suggestion
        Path for the template to use. If no path is provided, the default template is used.
```

File: src/pyconverter/xml2py/cli.py [hunk 3]
```diff
@@ -24,13 +24,13 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
     target_path: str, optional
         Path to the directory where you want the autogenerated package to be created.
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:25]:
```suggestion
        Path to the directory where you want to create the autogenerated package.
```
Avoid future tense where possible.

File: src/pyconverter/xml2py/cli.py [hunk 4]
```diff
@@ -24,13 +24,13 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
...
         The default value is the current working directory.
-    
+
     template_path: str, optional
         Path to the directory that will be used as a template.
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:29]:
```suggestion
        Path to the directory for the template to use.
```

File: src/pyconverter/xml2py/cli.py [hunk 5]
```diff
@@ -24,13 +24,13 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
...
-        The default value is the _package directory accessible in the 
+        The default value is the _package directory accessible in the
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:30]:
```suggestion
        The default value is the ``_package`` directory accessible in the
```

File: src/pyconverter/xml2py/writer.py [hunk 6]
```diff
@@ -239,14 +241,17 @@ def write_source(
...
     path_custom_functions : str, optional
         Path containing the customized functions. The default is ``None``.
 
-    new_package_path : str, optional
-        Path to copy the ``_package`` directory to. The default is ``./package``.
+    new_package_name : str, optional
+        Name of the new package. The default is ``package``.
+
+    template_path : str, optional
+        Path for the template to use. If no path is provided, the default template is used.
```
Comment (RobPasMue) [inline src/pyconverter/xml2py/writer.py:254]: Reorder. The function's definition has these arguments the other way around
```suggestion
    template_path : str, optional
        Path for the template to use. If no path is provided, the default template is used.

    new_package_name : str, optional
        Name of the new package. The default is ``package``.
```

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**